examples/src/RecordMovieComponent.tsx

```diff
@@ -28,16 +26,16 @@ const RecordMovieComponent = ({
     }, [isRecording]);
 
     const startRecording = async () => {
-        setRecordingStatus("Recording...");
+        if ("VideoEncoder" in window) {
         setOutputStatus("");
-        setIsRecording(true);
+        setIsRecording(true);}
```

Comment:
Nit: weird indentation here?

Reply:
removing this anyway and adding `broswerSupported` state to disable buttons with